### PR TITLE
Add command to list tags of current directory to Readme

### PR DESCRIPTION
**Pull request by timpulver**

### CODE REVIEW

**Comment by timpulver** [comment]
`cut` is not working here, no easy one-liner solution to split the various tags (tag1,tag2,tag3,…) found.

**Comment by timpulver** [comment]
Changed the split-command to `tr`

File: README.md [hunk 1]
```diff
@@ -121,7 +121,10 @@ You may use the --enter or --descend options to list the contents of, or recursi
     tag --list --enter .
     tag --list --descend .
     tag -d .
+    
+To get a list of all tags in the current directory you can combine `tag` with `tr` (to split multiple tags from one file to a seperate line), `sort`, and `uniq` (which removes duplicate entries):
 
+    tag --no-name * | tr "," "\n" | sort | uniq
```
Comment (jdberry) [inline README.md]: Note that you can accomplish the same thing more easily without using tr, by using the garrulous flag:

`tag -Ng * | sort | uniq`

Comment (jdberry) [inline README.md]: And the unique flag to sort also shortens your command:

`tag -Ng * | sort -u`

While this example shows how you can combine the usage of tag with various other shell tools, I'm not sure that it's appropriate for the Readme.

**Comment by timpulver** [comment]
Okay, good to know.
You’re right, putting piped cli-commands in the readme is probably not the best.
But I think it’s a very useful feature, so maybe this would would make a good addition (to readme or codebase).

**Comment by jdberry** [comment]
@timpulver I've decided not to add this anywhere for the being. Thanks.

**Pull request closed without merging**